test/docker/Test.dockerfile

```diff
@@ -31,6 +31,11 @@ RUN curl -OL https://dl.google.com/dl/cloudsdk/channels/rapid/install_google_clo
     bash install_google_cloud_sdk.bash --disable-prompts --install-dir='/root/' && \
     ln -s /root/google-cloud-sdk/bin/gcloud /usr/local/bin/gcloud
 
+# terraform
+RUN curl -sSL https://releases.hashicorp.com/terraform/0.12.10/terraform_0.12.10_linux_amd64.zip -o /tmp/tf.zip \
```

Comment:
setting the version with an `ENV` might be easier to find in case this file gets larger

Reply:
Thanks so much for a quick review. I've updated the config's file name and the terraform version.